packages/wallet-specs/src/ChannelStoreEntry.ts

```diff
@@ -1,5 +1,6 @@
-import { Channel, getChannelId, SignedState, State } from '.';
+import { Channel, getChannelId, SignedState } from '.';
```

Comment:
Minor but I think it would be nicer to import these from `@statechannels/nitro-protocol`